boxsdk/session/session.py

```diff
@@ -285,11 +286,14 @@ def _get_retry_after_time(self, attempt_number, retry_after_header):
         :return:                        Number of seconds to wait before retrying.
         :rtype:                         `Number`
         """
-        min_randomization = 1 - self._retry_randomization_factor
-        max_randomization = 1 + self._retry_randomization_factor
-        randomization = (random.uniform(0, 1) * (max_randomization - min_randomization)) + min_randomization
-        exponential = math.pow(2, attempt_number)
-        return exponential * self._retry_base_interval * randomization
+        try:
+            return int(retry_after_header)
```

Comment:
Do we want to just return this value, as-is?  I would think that we'd want to return the greater of:
- `return exponential * self._retry_base_interval * randomization`
- `int(retry_after_header)`

Also, is `retry_after_header` in units of seconds or is it a datetime or... ?

Reply:
Could this result in the time between retries not following the exponential backoff in the SDK?  

I'm not clear on how the server determines this value.  If the server provides a value, then it makes sense not to retry BEFORE that time.  But if there are multiple requests being rate-limited, we might want the variability that the SDK's backoff logic provides in order to spread out requests to avoid a Thundering Herd situation.

Reply:
I assume the server must calculate some variable time that request should wait for after a 429 since it is should be giving a time after which your service should not get another 429. I think the server must be properly calculating a time to avoid the Thundering Herd situation.

Also, the retry-after time is returned for other services like getting thumbnail representations. If we wait for the longer time, the thumbnail representation generation will be slower.